src/OpenLoco/OpenLoco.cpp

```diff
@@ -1024,10 +1024,10 @@ namespace OpenLoco
                     }
 
                     // clang-format off
-					if (today.month == MonthId::january ||
-						today.month == MonthId::april ||
-						today.month == MonthId::july ||
-						today.month == MonthId::october)
+					if (today.month == Month::January ||
+						today.month == Month::April ||
```

Comment:
personally I prefer table-style formatting (for more info on this style see here: https://alexastva.gitbooks.io/the-ultimate-question-of-programming-refactoring-/content/13.html)
if (today.month == Month::January
    || today.month == Month::April
    || today.month == Month::July
    || today.month == Month::October)